SEED-platform/seed
# 2226 lost data on import

#### Any background context you want to provide?
Data could potentially be lost on import (see attached issue).

#### What's this PR do?
Fixes a bug where multiple Column objects were being created for the same raw import file header. 

This caused multiple ColumnMapping objects to be created when only one was expected in order to initially create incoming import records.

#### How should this be manually tested?
Import a file twice. The first time, **don't** map incoming columns to unit-aware SEED columns. The second time, map to unit-aware columns. See that the data persists throughout import.

Import a file once, but map twice. The first time mapping, **don't** map incoming columns to unit-aware SEED columns. The second time (by clicking "Back to Mapping"), map to unit-aware columns. See that the data persists throughout import.

#### What are the relevant tickets?
#2226 

#### Screenshots (if appropriate)
